console/src/app/modules/user-grants/user-grants.component.html

```diff
@@ -30,7 +30,7 @@
 
   <a
     actions
-    *ngIf="disableWrite === false && selection.hasValue() === false"
+    *ngIf="disableWrite === false && selection.hasValue() === false && showAddButton"
```

Comment:
```suggestion
    *ngIf="disableWrite === false && selection.hasValue() === false && routerLink"
```

Reply:
lets use the `routerLink` variable itself to check if the button should display or not